src/employee.java

```diff
@@ -0,0 +1,31 @@
+public class employee {
+    String Name;
+    String Position;
+    String email;
+    String Phone;
+    int Salary;
+    int Age;
+    employee(String Name, String Position,String email, String Phone,int Salary, int Age ){
+        this.Name=Name;
+        this.Position=Position;
+        this.email=email;
+        this.Phone=Phone;
+        this.Salary=Salary;
+        this.Age=Age;
+    }
+public void PrintPersonalData(){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.